eng/pipelines/coreclr/pr.yml

```diff
@@ -30,176 +12,111 @@ jobs:
-      testGroup: innerloop
-
-#
-# Release builds
-#
-- template: /eng/pipelines/common/platform-matrix.yml
-  parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-job.yml
-    buildConfig: release
-    platforms:
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_rhel6_x64
-    - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
     - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
       testGroup: innerloop
 
 #
-# Checked test builds
+# Build CoreFX using live CoreLib from CoreCLR
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
+    jobTemplate: /eng/pipelines/libraries/build-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
-      testGroup: innerloop
+      isOfficialBuild: false
+      testScope: empty
+      useLiveCoreClr: true
 
 #
-# Checked JIT test executions
+# Test builds using live CoreFX
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/run-test-job.yml
+    jobTemplate: /eng/pipelines/coreclr/templates/build-test-job.yml
     buildConfig: checked
     platforms:
-    - Linux_arm
-    - Linux_arm64
-    - Linux_musl_x64
-    - Linux_x64
     - OSX_x64
-    - Windows_NT_arm
-    - Windows_NT_arm64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
-    helixQueueGroup: pr
-    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+    - Windows_NT_x64
+    - Windows_NT_arm64
     jobParameters:
       testGroup: innerloop
+      useLiveLibraries: true
 
 #
-# Checked R2R test executions
+# CoreCLR Test executions
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
     jobTemplate: /eng/pipelines/coreclr/templates/run-test-job.yml
     buildConfig: checked
     platforms:
-    - Linux_x64
     - OSX_x64
-    - Windows_NT_x64
+    - Linux_x64
     - Windows_NT_x86
+    - Windows_NT_x64
+    - Windows_NT_arm64
     helixQueueGroup: pr
     helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
     jobParameters:
       testGroup: innerloop
-      readyToRun: true
-      displayNameArgs: R2R
+      useLiveLibraries: true
 
 #
-# CoreFX test runs against CoreCLR
+# CoreFX Test Build
 #
 - template: /eng/pipelines/common/platform-matrix.yml
   parameters:
-    jobTemplate: /eng/pipelines/coreclr/templates/run-test-job.yml
+    jobTemplate: /eng/pipelines/libraries/build-test-job.yml
     buildConfig: checked
     platforms:
+    - OSX_x64
     - Linux_x64
+    - Windows_NT_x86
     - Windows_NT_x64
-    helixQueueGroup: pr
-    helixQueuesTemplate: /eng/pipelines/coreclr/templates/helix-queues-setup.yml
+    - Windows_NT_arm64
```

Comment:
Tests aren't arch specific, so we should be able to just build them for x64 in all OSs.